app/src/main/java/org/mozilla/fenix/components/bookmarks/BookmarksUseCase.kt

```diff
@@ -47,10 +47,13 @@ class BookmarksUseCase(storage: BookmarksStorage) {
          * Retrieves a list of recently added bookmarks, if any, up to maximum.
          */
         @WorkerThread
-        suspend operator fun invoke(count: Int = DEFAULT_BOOKMARKS_TO_RETRIEVE): List<BookmarkNode> {
+        suspend operator fun invoke(
+            count: Int = DEFAULT_BOOKMARKS_TO_RETRIEVE,
+            maxAge: Long = TimeUnit.DAYS.toMillis(DEFAULT_BOOKMARKS_DAYS_AGE_TO_RETRIEVE)
```

Comment:
super minor nit: because I've made the mistake of forgetting a param was in milliseconds. I found the [TabsUseCase](https://github.com/mozilla-mobile/android-components/blob/main/components/feature/tabs/src/main/java/mozilla/components/feature/tabs/TabsUseCases.kt#L348) naming format helpful.

```suggestion
            maxAgeInMs: Long = TimeUnit.DAYS.toMillis(DEFAULT_BOOKMARKS_DAYS_AGE_TO_RETRIEVE)
```